Policies/Kubernetes/container-no-privilege-escalation/azurepolicy.json

```diff
@@ -0,0 +1,55 @@
+{
+  "properties": {
+    "displayName": "Kubernetes PSP: Do not allow container privilege escalation in Kubernetes cluster",
```

Comment:
Can you remove Kubernetes PSP from display name? You can use category to filter PSP policies?